module/item/item.js

```diff
@@ -0,0 +1,626 @@
+import AbilityTemplate from "../pixi/ability-template.js";
+
+
+export default class ItemSg extends Item {
+
+    /* -------------------------------------------- */
+    /* Overrides                                    */
+    /* -------------------------------------------- */
+
+    /** @inheritdoc */
+    getRollData() {
+        let rollData = super.getRollData();
+
+        // Set the Tension Die from the scene, and if necessary, from the campaign
+        // Multiple different ways of writing to cover spelling errors
+        const tensionDie = game.sgrpg.getTensionDie();
+        rollData.tensionDie = tensionDie;
+        rollData.tensionDice = tensionDie;
+        rollData.TD = tensionDie;
+        rollData.td = tensionDie;
+        rollData.tD = tensionDie;
+
+        return rollData;
+    }
+
+    /** @override
+     * Augment the basic Item data model with additional dynamic data.
+     */
+    prepareDerivedData() {
+        // Get the Item's data
+        const itemData = this.data;
+        const actorData = this.actor ? this.actor.data : {};
+
+        // Bulk calculation
+        const data = itemData.data;
+        data.bulkTotal = 0;
+        if (data.bulk) {
+            data.bulkTotal = data.bulk * data.quantity;
+        }
+        if (data.quantity !== 1) data.isStack = true;
```

Comment:
I would say condition should be `quantity > 1` as i would not consider 0 to be stack for example